L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

```diff
@@ -2792,6 +2825,9 @@ bool l1t::TriggerMenuParser::parseAXOL1TL(L1TUtmCondition condAXOL1TL, unsigned
 
   (m_vecAXOL1TLTemplate[chipNr]).push_back(axol1tlCond);
 
+  //fill class model version variable
+  l1t::TriggerMenuParser::setAXOL1TLModelVersion(model);
```

Comment:
Why is this needed ?

For what I understand, there is one instance of `TriggerMenuParser`, and it has one variable `m_axol1tlModelVersion`, so if the L1T menu has multiple algos using different model versions, this would get set to just one of those versions (not clear which one). `TriggerMenuParser::m_axol1tlModelVersion` and `TriggerMenuParser::AXOL1TLModelVersion` do not seem to be used anywhere (which I guess makes sense, because the model version at this point is not a global property of the menu, but it's algo specific). So, what's the point of `TriggerMenuParser::AXOL1TLModelVersion` ?

Reply:
You're right. I forgot to remove this during the cleanup.

Reply:
There's actually more global model stuff left. Cleaning up everything now.